Source/Directory.Build.props

```diff
@@ -25,4 +27,9 @@
 		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
 		<NoWarn>NU5118</NoWarn>
 	</PropertyGroup>
+
+	<ItemGroup>
+		<SupportedPlatform Include="browser" />
```

Comment:
Should this be here? Fluxor.csproj is UI agnostic.

Reply:
All it is is a safeguard to ensure that no committer accidentally uses code marked with the following [attribute](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.versioning.unsupportedosplatformattribute)

```csharp
[System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

By putting this element into the Directory.Build.props you just ensure you will get a compiler error if anyone were to use a class/function decorated as such in one of their edits in all three of your libraries.

If you create a blazorwasm hosted project you can see that the Microsoft template puts the element in the csproj on the shared csproj even though it is not exclusively a WASM class library.

```
dotnet new blazorwasm -ho -o BlazorDemo
```